steps/fortify-on-demand-mobile-assessment/0.0.1/step.yml

```diff
@@ -0,0 +1,121 @@
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- client_id: null
+  opts:
+    description: Authenticating with Client Credentials. The client credentials are
+      the API keys generated in the portal.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: Client ID for authentication
+    title: Client ID
+- client_secret: null
+  opts:
+    description: Authenticating with Client Credentials. The client credentials are
+      the API keys generated in the portal.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: Client secret for authentication
+    title: Client Secret
+- datacenter: Select Datacenter
+  opts:
+    description: Select the datacenter assigned to you. This is provided once your
+      are onboarded with Fortify on Demand.
+    is required: true
+    is_expand: true
+    summary: Fortify on Demand Datacenters
+    title: Datacenter
+    value_options:
+    - Select Datacenter
+    - AMS
+    - APJ
+    - EMEA
+    - FED
+- entitlement_id: null
+  opts:
+    description: The ID assigned to ypur current entitlement. This value can be reterived
+      in Foritfy on Demand portal.
+    is_expand: true
+    is_required: true
+    summary: Entitlement ID for this mobile assessment
+    title: Entitlement ID
+- opts:
+    description: The ID assigned to the created release. This value can be reterived
+      in Foritfy on Demand portal.
+    is_expand: true
+    is_required: true
+    summary: Release ID for this mobile assessment
+    title: Release ID
+  release_id: null
+- assessment_type: Select Assessment Type
+  opts:
+    description: "Mobile Assessment includes Vulnerability analysis of mobile binary,
+      Endpoint reputation analysis and Security expert review of prioritized results.
+      \nMobile+ Assessment includes Manual testing of binary, network and services,
+      and DAST of backend services\nFor more information, please refer to the datasheet
+      at https://www.microfocus.com/media/data-sheet/fortify_on_demand_ds.pdf\n"
+    is required: true
+    is_expand: true
+    summary: Assessment type for this mobile assessment
+    title: Assessment Type
+    value_options:
+    - Select Assessment Type
+    - Mobile Assessment (Single Scan)
+    - Mobile+ Assessment (Single Scan)
+    - Mobile Assessment (Subscription)
+    - Mobile+ Assessment (Subscription)
+- framework_type: Select Framework Type
+  opts:
+    is required: true
+    is_expand: true
+    summary: Framework type for this mobile assessment
+    title: Framework Type
+    value_options:
+    - Select Framework Type
+    - Android
+    - iOS
+- opts:
+    is required: true
+    is_expand: true
+    summary: Platform type for this mobile assessment
+    title: Platform Type
+    value_options:
+    - Select Platform Type
+    - Phone
+    - Tablet
+    - Both
+  platform_type: Select Platform Type
```

Comment:
Please remove the "Select XYZ" type of value options and just leave these fields without default value.
If this is left this way, users will not realize that they have to pick one of the available options and the step will fail.

Reply:
I got any error if i didn't set the default

Reply:
oh :( That is unfortunate. In this case I think we should go with the initial "defaults".
I'll bring it to the team to be able to set no-defaults for the value options steps/

Reply:
Can i leave it blank like blow?

>   - platform_type: 
>     opts:
>       title: Platform Type
>       summary: Platform type for this mobile assessment
>       is_expand: true
>       is required: true
>       value_options:
>         - "Phone"
>         - "Tablet"
>         - "Both"

Reply:
Probably it will not pass the audit. This is something I was not aware of. Please select `AMS, Android, Phone` for default as of now. I'm sorry for the confusion.

Reply:
this is the error i got

![image](https://user-images.githubusercontent.com/44498335/134916998-73aea26d-c810-4e27-9891-36eff8bd27ec.png)

Reply:
Thank you. Just go with the initial solution.

Reply:
No worries. :)
I do agree with you on the initial values.
I have used " " as the initial values on the latest commit.
the audit pass

Reply:
Yes but in this way, a user who adds this step to their workflows will not be prompted to select one of the "available" values. They will be able to start the build and fail immediately because of the invalid selection. Maybe this behavior is better than preselecting one of the configurations.

Reply:
the latest commit reflect the changes.
please kindly review